backend/app.py

```
@@ -23,7 +23,7 @@ def login():
     hashed = hashlib.md5(password.encode("utf-8")).hexdigest()
     conn = get_db()
     cur = conn.cursor()
-    cur.execute(f"SELECT id FROM users WHERE username='{username}' AND password='{hashed}'")
+    cur.execute(f"SELECT id FROM users WHERE username='{username123}' AND password='{hashed}'")
```

Ruff (0.13.3)

Possible SQL injection vector through string-based query construction

(S608)